src/main/java/core/basesyntax/service/FileWriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FileWriterService {
+    void writeToFile(String filepath, String report);
```

Comment:
```suggestion
    void writeToFile(String filepath, String data);
```
this service can write any string value to a file, not just a report